profiles/OCI Object Storage us-ashburn-1.cyberduckprofile

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Protocol</key>
+	<string>s3</string>
+	<key>Vendor</key>
+	<string>oracle-us-phoenix-1</string>
```

Comment:
Vendor strings should be unique among all profiles.

Reply:
This is a remnant from my previous testing. Do we need Regions key at all? When it is omitted, does Cyberduck pick the region from Region key while creating top level folders (i.e. buckets)?